rmgpy/data/solvation.py

```diff
@@ -763,6 +752,18 @@ def load(self, path, libraries=None, depository=True):
 
         self.load_groups(os.path.join(path, 'groups'))
 
+    def get_solvent_critical_temperature(self, solvent_name):
+        """
+        Returns the critical temperature of the solvent in K for the given solvent_name.
+        The critical temperature is given by CoolProp function.
+        If the given solvent_name is not available in CoolProp, it raises DatabaseError.
+        """
+        try:
+            Tc = PropsSI('T_critical', solvent_name)
+        except:
+            raise DatabaseError(f"Critical temperature is not available for the given solvent: {solvent_name}")
+        return Tc
```

Comment:
Since this method is static and doesn't really relate to the database class, I would move this out as a standalone function inside of this file